qa/tasks/vstart_runner.py

```diff
@@ -232,7 +232,16 @@ def get_file(self, path, sudo, dest_dir):
         return tmpfile
 
     def put_file(self, src, dst, sudo=False):
-        shutil.copy(src, dst)
+        if sys.version_info.major < 3:
+            exception = shutil.Error
+        elif sys.version_info.major >= 3:
+            exception = shutil.SameFileError
+
+        try:
+            shutil.copy(src, dst)
+        # for the case where test code might copying a file on remote machine.
+        except exception:
```

Comment:
i believe you should just catch `EEXIST` yes? Blanket excepts should be rare.

Reply:
> i believe you should just catch `EEXIST` yes?

I don't see `EEXIST` occuring -
```
[rishabh@p50 ~]$ python
Python 2.7.15 (default, Oct 15 2018, 15:26:09) 
[GCC 8.2.1 20180801 (Red Hat 8.2.1-2)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import shutil
>>> shutil.copy('foo', 'foo')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/shutil.py", line 133, in copy
    copyfile(src, dst)
  File "/usr/lib64/python2.7/shutil.py", line 83, in copyfile
    raise Error("`%s` and `%s` are the same file" % (src, dst))
shutil.Error: `foo` and `foo` are the same file
>>> 
```

>  Blanket excepts should be rare.

Right. I've added the code in this update to raise the exception if `shutil.Error` was not raised due to same path in source and destination location.